README.md

```diff
@@ -45,7 +45,6 @@ webpage: https://www.epfl.ch/labs/cosmo/people/)
 git clone https://github.com/cosmo-epfl/chemiscope
 cd chemiscope
 npm install
-npm run download-example-input
```

Comment:
Why have we removed this? If you are doing dev it's really important you have the example input. Is it because we're moving the package to be installed more beyond dev purposes?

Reply:
No, this is because example input are now automatically downloaded when running `npm start` =)